embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/OpenTelemetrySdk.kt

```diff
@@ -16,11 +17,17 @@ internal class OpenTelemetrySdk(
     openTelemetryClock: Clock,
     spanProcessor: SpanProcessor
 ) {
+    private var resource: Resource = Resource.getDefault().toBuilder()
+        .put("service.name", BuildConfig.LIBRARY_PACKAGE_NAME)
```

Comment:
This works for now. We should bring in the semantic convention module as a dependency so we can import the names here rather than have custom hardcoded constants.